3f/pygmentize/2016-11-05.yaml

```diff
@@ -0,0 +1,9 @@
+title: Remote Code Execution
+link: https://github.com/dedalozzo/pygmentize/issues/1
+cve: ~
+branches:
+    master:
+        time: 2016-12-31 00:00:00
+        versions: ['<=1.1']
```

Comment:
@h4ckninja can you update this to `<1.2` then, as it is the version of the release containing the fix ?